src/backend/dx11/src/lib.rs

```diff
@@ -226,6 +226,19 @@ fn get_limits(feature_level: d3dcommon::D3D_FEATURE_LEVEL) -> hal::Limits {
         | _ => 8,
     };
 
+    // https://docs.microsoft.com/en-us/windows/win32/api/d3d11/nf-d3d11-id3d11device-checkmultisamplequalitylevels#remarks
+    // for more information.
+    let max_samples = match feature_level {
+        d3dcommon::D3D_FEATURE_LEVEL_9_1
+        | d3dcommon::D3D_FEATURE_LEVEL_9_2
+        | d3dcommon::D3D_FEATURE_LEVEL_9_3
+        | d3dcommon::D3D_FEATURE_LEVEL_10_0 => 0b0000_0001, // Conservative, MSAA isn't required.
+        d3dcommon::D3D_FEATURE_LEVEL_10_1 => 0b0000_1111, // Optimistic, 4xMSAA is required on all formats _but_ RGBA32.
```

Comment:
why are we optimistic here?
Also, is this the correct mask? I often get it wrong, but you can check the Metal example for the reference. Maybe this should be `1 | 4` instead of this value